rlm_python/radius_entrypoint.py

```diff
@@ -59,7 +58,11 @@ def setup_certs(
             sys.exit(1)
         if cert_ca != CERT_CA_DEST:
             print(f"Copying {cert_ca} to {CERT_CA_DEST}")
-            shutil.copyfile(cert_ca, CERT_CA_DEST)
+            try:
+                shutil.copyfile(cert_ca, CERT_CA_DEST)
+            except shutil.SameFileError:
+                pass
```

Comment:
We should error if we can't copy to the ca dest, else we'll just silently not have a CA available that the user expects.

Reply:
This is literally if it's the same file, so it's fine. It'll error if it's anything else.